### ISSUE TITLE
create an unlimited number of pieces from a filled cell with vertical bar representation

### ISSUE THREAD

**Issue by veillette** [phetsims/fractions-intro#63]
In vertical bar representation, clicking on a cell (without dragging) generate a piece that flies to the bucket. However the filled cell does not get toggle to empty, allowing it to be clicked again.

**Comment by coledu** [comment]
What I have learn thus far about this bug:

- Is that this bug only occur on the second pick up after the first pick up without any dragging.
- If you offset the center even by a small number this bug do not occur.
- That the cell color change back and forth after this bug happen which mean that the cell isFIlledProperty is begin toggling (I assume).
- That the specific container cell act as if it is filled. Reasons behind this is you increase the numerator the piece fly from the bucket to the next empty cell that is not the specific container cell. Also the next closest empty cell does not read the specific cell as empty. So it as if this cell is always filled.

A couple of line I have question thus far are:

Piece Drag Handler
```
if ( destinationCell.boundsProperty.value.containsPoint( piece.positionProperty.value ) ) {
            piece.cellToProperty.value = destinationCell;
          }
          else {
            piece.animateToAndFrom( piece.positionProperty.value, IntroConstants.BUCKET_POSITION );
```

- After the second pick up this if statement is always false.
- Must have something to do with the is Filled Property.

```options.startDrag();```

- We also look at this

Piece

```if ( distance > 0 ) {
        var animationTween = new TWEEN.Tween( location )
          .to( { x: finalPosition.x, y: finalPosition.y },
            distance * 5 )
          .easing( TWEEN.Easing.Cubic.InOut )
          .onUpdate( function() {
            self.positionProperty.value = new Vector2( location.x, location.y );
          } )
          .onComplete( function() {

            options.onComplete();

            // the piece can be removed from the pieces observable array
            self.reachedDestinationEmitter.emit();
          } );

        animationTween.start( phet.joist.elapsedTime );
      }
      else {

        // for cases where the distance is zero

        //  optional call back
        options.onComplete();

        // the piece can be removed from the pieces observable array
        self.reachedDestinationEmitter.emit();
```

- This is the animate to and from in piece which might have something to do with the centerpostion of the piece begin right on top of the final position so the distance is 0 which cause us to do the else statement.

So in conclusion this bug most likely have something to do with the isFIlledProperty of that cell. But I still do not understand have the centerPostion or if we drag it affect this bug.

**Comment by coledu** [comment]
The bug was an unnecessary if statement. Took it out and it fixed the bug. Assigning to @veillette for review.

**Comment by veillette** [comment]
This issue is stale since we used @jonathanolson prototype.
